libs/partners/elasticsearch/pyproject.toml

```diff
@@ -29,6 +29,7 @@ pytest-asyncio = "^0.21.1"
 langchain = { path = "../../langchain", develop = true }
 langchain-community = { path = "../../community", develop = true }
```

Comment:
does the factoring out of text splitters allow us to remove either of these?

Reply:
Indeed, community can be removed here since it's pulled in via `langchain` anyways. Added a PR: https://github.com/langchain-ai/langchain/pull/18644

(We need `langchain` for the chat history test.)
https://github.com/langchain-ai/langchain/blob/81985b31e6ff839ba073d1107722e1756d30e678/libs/partners/elasticsearch/tests/integration_tests/test_chat_history.py#L7